xbmc/platform/win32/WIN32Util.cpp

```diff
@@ -312,6 +312,17 @@ int CWIN32Util::GetDesktopColorDepth()
 #endif
 }
 
+size_t CWIN32Util::GetSystemMemorySize()
+{
+#ifdef TARGET_WINDOWS_STORE
+  return 0;
```

Comment:
Maybe we can use https://docs.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-globalmemorystatusex for uwp? It might not be exactly the same but better than not having anything?

Reply:
I like the idea for UWP. Indeed, I had not looked for the substitute enough 😄 

PR just updated.  On UWP reports 15.8 GB because is memory available to SO (not physically installed).